src/private/files.ts

```diff
@@ -226,6 +226,43 @@ export namespace files {
     listUrl?: string;
   }
 
+  /**
+   * @private
+   * Hide from docs
+   *
+   * Download Files interface
+   */
+  export interface IFileItem {
+    /**
+     * ID of the file metadata
+     */
+    objectId?: string;
+    /**
+     * Path of the file
+     */
+    path?: string;
+    /**
+     * Size of the file
+     */
+    size?: number;
+    /**
+     * Download status
+     */
+    status?: string;
```

Comment:
>status?: string; [](http://example.com/codeflow?start=4&length=16)

If this is a limited set of values, please use a type or enum so that callers can see what the potential status values are. Just using the string type doesn't give callers much information

Reply:
Comment Incorporated. Converted string to enum.